src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -101,7 +101,22 @@ func (r *ConsoleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 }
 
 func (r *ConsoleReconciler) createIngress(ctx context.Context, cluster *redpandav1alpha1.Cluster, console *redpandav1alpha1.Console, log logr.Logger) error {
-	return nil
+	subdomain := ""
+	proxyAPIExternal := cluster.PandaproxyAPIExternal()
```

Comment:
Why Pandaproxy? I would not couple console CR with cluster CR that way. If user would like to create 2 consoles that connects to single cluster it will not be able to achieve that.

I would add subdomain configuration filed into console CRD.

Reply:
Added `spec.subdomain`